src/test/index.js

```diff
@@ -10,13 +10,20 @@ const TASKS = [{
 }, {
   title: 'Test Browser',
   task: (opts) => {
+    if (opts.eek) {
```

Comment:
what is `opts.eek`? please use more descriptive identifiers

Reply:
This is all behind a flag and will be either deleted in the future or used by default so all this is going way.

https://github.com/ipfs/aegir/blob/1c0a5a1cd8846f3e2c93d30691f65b5d89381727/cmds/test.js#L61-L65